packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.js

```diff
@@ -1,9 +1,11 @@
 import _ from 'underscore';
 import moment from 'moment';
+import { send as sendEmail, checkEmail } from 'meteor/rocketchat:mailer';
```

Comment:
Let's nos use "as sendEmail" and keep the names the same everywhere for clarity.